### PR TITLE
add inverse scale factor import option

### PR DESCRIPTION
**Pull request by Yagich**
adds a new import option, "inverse scale factor", a la Qodot, that allows to change the scale per import. defaults to 32 just like the original constant.

### CODE REVIEW

**Review by jitspoe — APPROVED** [review]
Undecided if this should be just a simple scale or an inverse scale, but having the option is useful in any case!

File: addons/bsp_importer/bsp_reader.gd [hunk 1]
```diff
@@ -252,6 +251,13 @@ var plane_normals : PackedVector3Array
 var plane_distances : PackedFloat32Array
 var model_scenes : Dictionary = {}
 
+var _unit_scale: float = 1.0
+
+var inverse_scale_fac: float = 16.0:
```
Comment (jitspoe) [inline addons/bsp_importer/bsp_reader.gd:256]: Probably doesn't matter, but perhaps this should be the default 32.

**Comment by jitspoe** [comment]
Oops, I recently submitted some changes, would you mind merging?

File: addons/bsp_importer/bsp_reader.gd (hunk 1 above)
Comment (Yagich) [reply addons/bsp_importer/bsp_reader.gd:256]: oh good catch! i just copied it over from my code and missed this one

**Pull request merged**